src/main/java/bio/terra/cli/service/WorkspaceManagerService.java

```diff
@@ -332,7 +351,8 @@ public void grantIamRole(UUID workspaceId, String userEmail, IamRole iamRole) {
           () -> {
             new SamService(server, user).inviteUser(userEmail);
             return null;
-          });
+          },
+          (ex) -> isRetryable(ex) || SamService.isRetryable(ex));
```

Comment:
Isn't the SamService already doing retries internally for `inviteUser`, so why do we need to expose and use SamService.isRetryable?